### PR TITLE
Tab content depends on forest type availability

### PR DESCRIPTION
**Pull request by angsherpa232**

### CODE REVIEW

File: src/components/ProjectionResult.js [hunk 1]
```diff
@@ -7,9 +7,20 @@ import { setRecommendationMode } from '../store/actions';
...
+const requiredFieldsForProjection = [
+  'forestEcoregion',
+  'forestType',
+  'altitudinalZone',
+];
+
 function ProjectionResult() {
   const dispatch = useDispatch();
-  const projectionLocation = useSelector(state => state.projectionLocation);
+  const { location } = useSelector(state => ({
```
Comment (friedjoff) [inline src/components/ProjectionResult.js]:
```suggestion
  const location = useSelector(state => state.location);
```

File: src/components/ProjectionResult.js [hunk 2]
```diff
@@ -7,9 +7,20 @@ import { setRecommendationMode } from '../store/actions';
 import Recommendation from './Recommendation';
 import styles from './ProjectionResult.module.css';
 
+const requiredFieldsForProjection = [
```
Comment (friedjoff) [inline src/components/ProjectionResult.js]:
```suggestion
const requiredFieldsForResult = [
```

File: src/components/Recommendation.js [hunk 3]
```diff
@@ -25,7 +25,9 @@ function Recommendation({ futureDisabled }) {
     recommendationMode === 'today'
       ? location.forestType
       : projectionLocation.forestType;
-  const forestTypeLabel = translate('forestType', forestType, i18n.language);
+  const forestTypeLabel =
+    projectionLocation.forestType &&
```
Comment (friedjoff) [inline src/components/Recommendation.js]: Do we need this?

Comment (angsherpa232) [reply src/components/Recommendation.js]: Yes, otherwise it will throw an error.

**Pull request merged**